src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -2078,6 +2080,24 @@ public void Dispose ()
 				StringAssert.Contains ($"warning XA4", builder.LastBuildOutput, "warning XA4212");
 			}
 		}
+
+		[Test]
+		public void RunXABuildInParallel ()
+		{
+			var xabuild = new ProjectBuilder ("temp/RunXABuildInParallel").XABuildExe;
+			Parallel.For (0, 10, i => {
+				using (var p = Process.Start (new ProcessStartInfo (xabuild, "/version") {
```

Comment:
For future reference, I don't like overly complicated `using` block here. In particular, `)}} {` makes my eyes bleed. :-D

I love `using` blocks, and I love collection initializers, but I don't love intermixing them. I would prefer:

```csharp
var psi = new ProcessStartInfo (xabuild, "/version") {
    CreateNoWindow = true,
    // ...
};
using (var p = Process.Start (psi)) {
    // ...
}
```